src/clients/python/tests/test_basic.py

```diff
@@ -5,7 +5,7 @@
 import pytest
 
 import tigerbeetle as tb
-
+tb.configure_logging(debug=True)
```

Comment:
Since logging is per application and not per client, I though this was the most suitable API. (`debug=True` when creating a client would be nice, but not really what's happening behind the scenes).